examples/bri-3/src/bri/zkp/components/merkleTree/merkleTree.ts

```diff
@@ -0,0 +1,74 @@
+import { Tree } from "./tree"
+import { IMerkleTree} from "./merkleTree.interface";
+import { PreciseProofs } from "ew-precise-proofs-js";
+
+export class MerkleTree implements IMerkleTree {
+  tree: Tree;
+
+  constructor(document: any){
+    const merkleTree: Tree = this.createTree(document);
+
+    this.tree.leaves = merkleTree.leaves;
+    this.tree.root = merkleTree.root; 
+  }
+
+  private createTree(document: any): Tree {
```

Comment:
please add early exit if document is empty. Also, is there a specific reason to use "any" type for document?

Reply:
Currently, the Merkle tree library I am using supports documents in JSON format and is projected to support other formats in the future. Hence, I have kept the document type as "any".